scripts/subscribeEthereumDeposit.js

```diff
@@ -0,0 +1,149 @@
+
+const { Api } = require('@cennznet/api');
+require("dotenv").config();
+const { Keyring } = require('@polkadot/keyring');
+const logger = require('./logger');
+const mongoose = require('mongoose');
+const { BridgeClaim  } = require('../src/mongo/models');
+const { ethers } = require("hardhat");
+let txExecutor;
+const { curly } = require("node-libcurl");
+
+const airDropAmount = 50000;
+
+async function airDrop(claimId, signer, api, spendingAssetId) {
+    const signerBalance = await api.query.genericAsset.freeBalance(spendingAssetId, signer.address);
+    if (signerBalance.toNumber() > airDropAmount) {
+        const record = await BridgeClaim.findOne({claimId});
+        const cennznetAddress = record.cennznetAddress;
+        logger.info(`Air drop in progress for address ${cennznetAddress}`);
+        await api.tx.genericAsset.transfer(spendingAssetId, cennznetAddress, airDropAmount).signAndSend(signer);
```

Comment:
cap airdrop to once per address max.
shouldn't be able to get tokens for multiple claims from same account

Reply:
done.